### ISSUE TITLE
About BatchJob downloadBatchJobResults

### ISSUE THREAD

**Issue by kim-uen** [googleads/googleads-php-lib#215]
Hello,
About BatchJobsDelegate, I have one more troubles.
Its trouble is that  the result of  the method of downloadBatchJobResults is empty at all time.

I  investigated　the downloadBatchJobResults ;

```
    $mutateResponse = $this->batchJobSerializer->decode(
        $response->getBody()->getContent(),  'xml');

```
This result is empty.

It does not work well on  "$response->getBody()->getContent()" .
Alternatively,  When I changed this code to `(string)$response->getBody()`, It does work well .

Is there any another way that it work well simply?
My Environment is php 7.1.0

**Comment by fiboknacky** [comment]
Hello @kim-uen 

Thanks for reporting.
This is also be fixed with #197 as well.
In short, in some cases, when the stream of response is read until the end, it's not properly rewound into the beginning.

What you've done `(string) $response->getBody()` should work as well as `_toString` of Guzzle `StreamInterface` also does the rewinding for us.

Best,
Knack

**Comment by kim-uen** [comment]
Hello @fiboknacky 

Thank you for detail information.
I undestand thre reason of this problem.
I will try again with next release.

Best Regards
Kim
